### ISSUE TITLE
Unable to Sync Inventory Source - Ansible Tower

### ISSUE THREAD

**Issue by ryan012880** [ansible/awx#7109]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
An inventory configured with inventory source Ansible Tower is unable to sync

##### ENVIRONMENT
* AWX version: 11.2.0
* AWX install method: docker on linux
* Ansible version:  2.9.9
* Operating System: RHEL7
* Web Browser: Chrome

##### STEPS TO REPRODUCE

NOTE: need to have an existing AWX node that hosts the inventory data

1. Pull the latest commit of AWX
2. build a local AWX container image
3. Create an Ansible Tower kind credential to be used in the inventory source
4. Create an inventory with inventory source Ansible Tower, connect to the existing AWX

##### EXPECTED RESULTS

AWX should be able to sync host and groups from inventory source from Ansible Tower

##### ACTUAL RESULTS
```
    2.579 INFO     Updating inventory 2: FO
    3.167 INFO     Reading Ansible inventory source: /tmp/awx_11578_f30pyq46/tower.yml
    3.171 INFO     Using VIRTUAL_ENV: /var/lib/awx/venv/ansible
    3.171 INFO     Using PATH: /var/lib/awx/venv/ansible/bin:/var/lib/awx/venv/awx/bin:/usr/pgsql-10/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
    3.171 INFO     Using PYTHONPATH: /var/lib/awx/venv/ansible/lib/python3.6/site-packages:
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/bin/awx-manage", line 8, in <module>
    sys.exit(manage())
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/awx/__init__.py", line 154, in manage
    execute_from_command_line(sys.argv)
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/awx/main/management/commands/inventory_import.py", line 1139, in handle
    raise exc
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/awx/main/management/commands/inventory_import.py", line 1029, in handle
    venv_path=venv_path, verbosity=self.verbosity).load()
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/awx/main/management/commands/inventory_import.py", line 215, in load
    return self.command_to_json(base_args + ['--list'])
  File "/var/lib/awx/venv/awx/lib/python3.6/site-packages/awx/main/management/commands/inventory_import.py", line 198, in command_to_json
    self.method, proc.returncode, stdout, stderr))
RuntimeError: ansible-inventory failed (rc=1) with stdout:
stderr:
ansible-inventory 2.9.9
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/var/lib/awx/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible-inventory
  python version = 3.6.8 (default, Nov 21 2019, 19:31:34) [GCC 8.3.1 20190507 (Red Hat 8.3.1-4)]
Using /etc/ansible/ansible.cfg as config file
[WARNING]:  * Failed to parse /tmp/awx_11578_f30pyq46/tower.yml with auto
plugin: inventory config '/tmp/awx_11578_f30pyq46/tower.yml' specifies unknown
plugin 'awx.awx.tower'
  File "/usr/lib/python3.6/site-packages/ansible/inventory/manager.py", line 280, in parse_source
    plugin.parse(self._inventory, self._loader, source, cache=cache)
  File "/usr/lib/python3.6/site-packages/ansible/plugins/inventory/auto.py", line 53, in parse
    raise AnsibleParserError("inventory config '{0}' specifies unknown plugin '{1}'".format(path, plugin_name))
[WARNING]: Unable to parse /tmp/awx_11578_f30pyq46/tower.yml as an inventory
source
ERROR! No inventory was parsed, please check your configuration and options.
```

**Comment by ryanpetrello** [comment]
@AlanCoding @kdelee is this a duplicate of https://github.com/ansible/awx/issues/6878?

**Comment by AlanCoding** [comment]
probably, we have a few similar reports right now

**Comment by ryan012880** [comment]
btw I've tried up to the below commit only and it is working:

commit 1ca3d1fe3bb5bd6c30d49ac50c7ee134064c8edd
Merge: 6b8359f 8cdffd0
Author: softwarefactory-project-zuul[bot] <33884098+softwarefactory-project-zuul[bot]@users.noreply.github.com>
Date:   Mon May 11 15:33:07 2020 +0000

**Comment by MarcoV-git** [comment]
Guys, it seems that this is still not fixed in 12.0.0
Both with Satellite and VMWare I get:

    RuntimeError: ansible-inventory failed (rc=1) with stdout:
    stderr:
    ansible-inventory 2.9.9
      config file = /etc/ansible/ansible.cfg
      configured module search path = ['/var/lib/awx/.ansible/plugins/modules',  '/usr/share/ansible/plugins/modules']
      ansible python module location = /usr/lib/python3.6/site-packages/ansible
      executable location = /usr/bin/ansible-inventory
      python version = 3.6.8 (default, Nov 21 2019, 19:31:34) [GCC 8.3.1 20190507 (Red Hat 8.3.1-4)]
    Using /etc/ansible/ansible.cfg as config file
    [WARNING]:  * Failed to parse /tmp/awx_1186_n9l_s3qr/foreman.yml with auto
    plugin: inventory config '/tmp/awx_1186_n9l_s3qr/foreman.yml' specifies unknown
    plugin 'theforeman.foreman.foreman'
      File "/usr/lib/python3.6/site-packages/ansible/inventory/manager.py", line 280, in parse_source
        plugin.parse(self._inventory, self._loader, source, cache=cache)
      File "/usr/lib/python3.6/site-packages/ansible/plugins/inventory/auto.py", line 53, in parse
        raise AnsibleParserError("inventory config '{0}' specifies unknown plugin '{1}'".format(path, plugin_name))
    [WARNING]: Unable to parse /tmp/awx_1186_n9l_s3qr/foreman.yml as an inventory source
    ERROR! No inventory was parsed, please check your configuration and options.

And with VMware:

    [WARNING]:  * Failed to parse /tmp/awx_1185__0z02k4e/vmware_vm_inventory.yml
    with auto plugin: inventory config
    '/tmp/awx_1185__0z02k4e/vmware_vm_inventory.yml' specifies unknown plugin
    'community.vmware.vmware_vm_inventory'

Is there a specific reason why this is still broken in 12.0.0?

**Comment by ryan012880** [comment]
I manage to try the official image release of version 12.0.0 also the latest devel local image build and It's working! Thank you
